packages/types/src/lib/models.ts

```diff
@@ -0,0 +1,170 @@
+import { SiweMessage } from 'siwe';
+import { AuthSig } from './interfaces';
+
+export type PlainJSON =
+  | boolean
+  | number
+  | string
+  | { [key: string]: PlainJSON }
+  | Array<PlainJSON>;
```

Comment:
Will do!

Reply:
actually you can't, cus  `Type alias 'PlainJSON' circularly references itself.`
<img width="576" alt="image" src="https://github.com/LIT-Protocol/js-sdk/assets/4049673/9d454249-9144-439e-9d03-87310b58e851">

Reply:
ah i see, my worry with the `{ [key: string]: PlainJSON }` is that it will always require `['']` access syntax and we cannot us `.` notation on the object. but i think with a Record type we are able to use `.` notation.